paper-tabs.html

```diff
@@ -390,6 +420,62 @@
         }, 1);
       },
 
+      _activateHandler: function() {
+        this._cancelPendingActivation();
+        Polymer.IronMenuBehaviorImpl._activateHandler.apply(this, arguments);
+      },
+
+      _delayedActivationHandler: function() {
+        var item = this._pendingActivationItem;
+        this._pendingActivationItem = undefined;
+        this._pendingActivationTimeout = undefined;
+        item.dispatchEvent(new CustomEvent(this.activateEvent, {
+          bubbles: true,
+          cancelable: true
+        }));
+      },
+
+      _scheduleActivation: function(item, delay) {
+        this._pendingActivationItem = item;
+        this._pendingActivationTimeout = this.async(
+          this._bindDelayedActivationHandler, delay);
+      },
+
+      _cancelPendingActivation: function() {
+        if (this._pendingActivationTimeout !== undefined) {
+          this.cancelAsync(this._pendingActivationTimeout);
+          this._pendingActivationItem = undefined;
+          this._pendingActivationTimeout = undefined;
+        }
+      },
+
+      _onLeftKey: function(event) {
+        Polymer.IronMenubarBehaviorImpl._onLeftKey.apply(this, arguments);
+        if (!this.noAutoselect && this.autoselectDelay < 0) {
+          this._scheduleActivation(this.focusedItem, this.autoselectDelay);
+        }
+      },
+
+      _onRightKey: function(event) {
+        Polymer.IronMenubarBehaviorImpl._onRightKey.apply(this, arguments);
+        if (!this.noAutoselect && this.autoselectDelay < 0) {
+          this._scheduleActivation(this.focusedItem, this.autoselectDelay);
+        }
+      },
+
+      _onKeyup: function(event) {
```

Comment:
Can't we just use an a11yKeysBehaviour and have `keyBindings` for just the arrows?